src/BasicRum/DiagramSchema/Filters/Primary/DeviceType.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\BasicRum\DiagramSchema\Filters\Primary;
+
+class DeviceType implements \App\BasicRum\DiagramSchema\Filters\FilterableInterface
+{
+    public function getSchema(): array
+    {
+        $schema = [
+            'device_type' => [
+                'type' => 'object',
+                'properties' => [
+                    'search_value' => [
+                        'enum' => [1, 2, 3, 4, 5],
```

Comment:
Hello @korzol 

Di you think that the list of values here may change in the future? I think that we need to keep in mind this and fetch the values dynamically.